nvme-print.h

```diff
@@ -83,6 +83,8 @@ struct print_ops {
 	void (*lba_range)(struct nvme_lba_range_type *lbrt, int nr_ranges);
 	void (*lba_status_info)(__u32 result);
 	void (*d)(unsigned char *buf, int len, int width, int group);
+	void (*show_init)(void);
+	void (*show_finish)(void);
```

Comment:
and pass it into the function again...

but this might be  unnecessary complex thing to do or do we expect multithreading? so just have a init/finish call would be enough. let me ponder on this.

Reply:
Just I thought no opaque pointer returning and passing seem simple for the caller to use the APIs. Sure if needed I can change them as mentioned.

Reply:
Basically this means this API is not thread safe, which is fine for nvme-cli. I think we just go with your version, but maybe just call it the function `init` and `finish`.